### PR TITLE
Add top level import ability

### PR DESCRIPTION
**Pull request by juliusgeo**

### CODE REVIEW

File: changelog.rst [hunk 1]
```diff
@@ -0,0 +1,7 @@
+=========
+Changelog
+=========
+
+Changes in version 1.1.0
+
+- ``ExplainCollection`` now importable from top level like so: ``from pymongoexplain import ExplainCollection``
```
Comment (ShaneHarvey) [inline changelog.rst:7]: Can you add a changelog entry for 1.0.0?

File: pymongoexplain/__init__.py [hunk 2]
```diff
@@ -11,3 +11,5 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+from pymongoexplain.explainable_collection import ExplainCollection
```
Comment (ShaneHarvey) [inline pymongoexplain/__init__.py:15]: Newline

File: changelog.rst [hunk 3]
```diff
@@ -0,0 +1,7 @@
+=========
+Changelog
+=========
+
+Changes in version 1.1.0
```
Comment (ShaneHarvey) [inline changelog.rst:5]: Can you make this a heading:
```
Changes in version 1.1.0
------------------------
```

Comment (juliusgeo) [reply changelog.rst:5]: Done.

File: pymongoexplain/__init__.py (hunk 2 above)
Comment (juliusgeo) [reply pymongoexplain/__init__.py:15]: Done.

File: changelog.rst (hunk 1 above)
Comment (juliusgeo) [reply changelog.rst:7]: Done.

File: changelog.rst [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+------------------------
+- ``ExplainCollection`` now importable from top level like so: ``from pymongoexplain import ExplainCollection``
+
+Changes in version 1.0.0
+------------------------
+= Initial ``ExplainCollection`` API
```
Comment (prashantmital) [inline changelog.rst:11]: Should this be `=` or `-`?

Comment (juliusgeo) [reply changelog.rst:11]: Oops, it should be == because "changelog" is == on both top and bottom.

Comment (ShaneHarvey) [reply changelog.rst:11]: Prashant was talking about the `= Initial ``ExplainCollection`` API ` which needs to be `- Initial...`

File: pymongoexplain/__init__.py (hunk 2 above)
Comment (ShaneHarvey) [inline pymongoexplain/__init__.py:15]: There should be a test for this.

File: changelog.rst (hunk 4 above)
Comment (juliusgeo) [reply changelog.rst:11]: Done

File: pymongoexplain/__init__.py (hunk 2 above)
Comment (juliusgeo) [reply pymongoexplain/__init__.py:15]: This is addressed in: https://github.com/mongodb-labs/pymongoexplain/pull/36/commits/50778648da4480a413a65bb47f8f0129bd024e2d

File: changelog.rst [hunk 5]
```diff
@@ -0,0 +1,11 @@
+=========
+Changelog
+=========
+
+Changes in version 1.1.0
+------------------------
+- ``ExplainCollection`` now importable from top level like so: ``from pymongoexplain import ExplainCollection``
+
+Changes in version 1.1.0
```
Comment (ShaneHarvey) [inline changelog.rst:9]: 1.1.0 -> 1.0.0

Comment (prashantmital) [reply changelog.rst:9]: Ah, good catch

Comment (juliusgeo) [reply changelog.rst:9]: Done.

**Pull request merged**